### PR TITLE
Updated smolCwypto and added smolCwypto test cases

### PR DESCRIPTION
**Pull request by nguyenj21**
Issue Number: #58

Description: I just deleted the old test cases that weren't working, fixed the smolCwypto test cases by adding module export in smolCwypto.js, and making sure the import works.

Testing: smolCwypto should be able to create a valid hash and deal with empty data

Possible Errors: the files might have merge conflict?

Image:
image
image

### CODE REVIEW

File: small-talk/util/smolCwypto.test.js
Comment (blake-gilmore) [inline small-talk/util/smolCwypto.test.js:1]: Can you delete the existing smolCwyptoTEST file with this PR?

Comment (nguyenj21) [reply small-talk/util/smolCwypto.test.js:1]: Just deleted the existing smolCwyptotTEST

**Review by junxjr — APPROVED** [review]
Everything looks good to me. Great work on making the test 100%.

File: small-talk/util/smolCwypto.test.js
Comment (blake-gilmore) [reply small-talk/util/smolCwypto.test.js:1]: Thanks!

**Review by blake-gilmore — APPROVED** [review]
No issues here! LGTM

**Pull request merged**